### DISCUSSION TITLE
Org chart?

### DISCUSSION THREAD

**Discussion by daveyyydave** [microsoft-search/pnp-modern-search#2693] [Q&A]
Hello, shout out to the devs for creating such a wonderful search for Sharepoint.  Question...Is it possible to display the org chart as the result?

**Comment by kasperbolarsen** [comment]
I doubt it, but try to have a look at https://sharepointmaven.com/8-ways-to-display-employee-directory-in-sharepoint/

Are there some abilities you are missing in those solutions?

**Reply by daveyyydave** [reply]
> I doubt it, but try to have a look at https://sharepointmaven.com/8-ways-to-display-employee-directory-in-sharepoint/
> 
> Are there some abilities you are missing in those solutions?

Ideally, I want to display results of the search (which is what i have currently working), but also have maybe a org button or just org chart being displayed for the person I selected?

**Reply by kasperbolarsen** [reply]
Like the org tab in the Hover Card? Which can be customized as per https://learn.microsoft.com/en-us/graph/toolkit/components/person-card

**Reply by kasperbolarsen** [reply]
@daveyyydave did you found a solution matching the requirements?

**Reply by daveyyydave** [reply]
sorry, was busy with other projects, i have not yet.  i might just create a powerapp and embed it into webparts.
